material-overrides/assets/stylesheets/tanssi.css

```diff
@@ -1342,4 +1342,57 @@ h2.subtitle.mobile-subtitle {
   border-radius: 1em;
   vertical-align: baseline;
   box-shadow: none;
+}
+
+/* Cards Styling */ 
+.md-typeset .grid.cards>ul>li {
+  border: 1px solid #FFFAFA;
+  background-color: #3D3D3D;
+}
+.md-typeset .grid.cards>ul>li:hover {
+  border: 1px solid #189A9A;
+  background-color: #1B1B1B;
+}
+.md-typeset a:hover {
+  transform: translateX(10px);
+  transition: 0.3s ease;
+}
+.grid.cards > ul > li:hover >p:first-child {
+  transform: translateX(10px);
+  transition: 0.3s ease;
+}
+.md-typeset hr {
+border: o solid 
+}
+.md-typeset .grid {
+  grid-gap: 1rem;
+}
+.md-typeset .grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(min(100%, 16rem), 1fr));
+    margin: 1em 0;
+}
```

Comment:
These are both targeting `.md-typeset .grid`. But it looks like these aren't necessary because these are the same as the default values provided by material, so we should be able to delete these and the cards would look the same